process-automation/latest/model/forms.md

```diff
@@ -301,6 +301,44 @@ The advanced properties for a text field are:
 | Reversed | This reverses the entry for an `Input mask` and reads the text from right to left instead. |
 | Input mask placeholder | The placeholder to demonstrate the format of an `Input mask`. For example `(__) ____-____` in the phone number example. |
 
+### Metadata viewer
+
+The Metadata viewer can be used to view the metadata of any file you upload or attach to your GUI using the Attach file widget.  
+
+To create a form that contains the Metadata viewer:
+
+1. Create or edit an existing from, for more see [Create a form](#create-a-form).
+
+2. Add the Attach file widget to the form.
+
+3. Add the Metadata viewer widget to the form.
+
+4. Add the File viewer widget to the form.
```

Comment:
This step is not needed, The metadata viewer requires the attach widget to work but not the file viewer. THey usually appear together but it is not required